src/components/Textarea/Textarea.tsx

```diff
@@ -1,16 +1,24 @@
-import React, { TextareaHTMLAttributes, VFC, useEffect, useRef, useState } from 'react'
+import React, { TextareaHTMLAttributes, VFC, useCallback, useEffect, useRef, useState } from 'react'
 import styled, { css } from 'styled-components'
 
 import { Theme, useTheme } from '../../hooks/useTheme'
 import { useClassNames } from './useClassNames'
 
+import { defaultHtmlFontSize } from '../../themes/createFontSize'
+
 type Props = {
   /** 入力値にエラーがあるかどうか */
   error?: boolean
   /** コンポーネントの幅 */
   width?: number | string
   /** 自動でフォーカスされるかどうか */
   autoFocus?: boolean
+  /** 自動で広がるかどうか */
+  autoGrowable?: boolean
```

Comment:
nits) 英語力の問題かもしれませんが言葉的に馴染みがなく、`autoResize`や`autoResizeY`のほうがイメージしやすかったのですがいかがでしょうか？
```suggestion
  autoResize?: boolean
```

Reply:
修正しました!
https://github.com/kufu/smarthr-ui/pull/2598/commits/ce1f9e1ecceb5083ceb3c519fb725dbc193efd4e